indexing-service/src/main/java/org/apache/druid/indexing/common/task/KillUnusedSegmentsTask.java

```diff
@@ -66,6 +72,20 @@
  * The client representation of this task is {@link ClientKillUnusedSegmentsTaskQuery}.
  * JSON serialization fields of this class must correspond to those of {@link
  * ClientKillUnusedSegmentsTaskQuery}, except for {@link #id} and {@link #context} fields.
+ * <br/>
```

Comment:
Thanks for the javadoc.
Tip, you can use `<p>` to separate paragraphs. It looks nicer.

Reply:
Done, thanks